addresspoints/README.md

```diff
@@ -37,3 +37,8 @@ And then run the service linking to the previous container:
 `docker run --rm --name grasshopper-addresspoints -e ELASTICSEARCH_HOST=192.168.59.103 -e ELASTICSEARCH_PORT=9300 -p 8081:8081 --link elasticsearch:elasticsearch hmda/grasshopper-addresspoints`
 
 The Elasticsearch host and port are configurable, passing them as environment variables to the docker container. If not specified, the defaults are _localhost_ and _9300_, respectively.
+
+Additionally, runtime monitoring metrics can be sent to a StatsD host by passing the following additional environment variable:
+
+`-e STATSD_HOST=192.168.59.103` (change value as appropriate for your environment)
```

Comment:
...along this same line, should this PR include a StatsD Docker image?  Unfortunately, it doesn't looks like there's heavily used image that's _just_ StatsD.  From what I see up there, I like the [shakr/statsd-influxdb](https://registry.hub.docker.com/u/shakr/statsd-influxdb/) image best.  It seems quite configurable.  If we decide to build our own, we may want to follow their lead.

...and while we're at it, [Grafana](https://registry.hub.docker.com/u/grafana/grafana/)?